index.html

```diff
@@ -51,6 +51,7 @@ <h2 class="text-center text-3xl font-semibold text-indigo-100">
 
         <div
           class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-4 gap-4"
+          onclick="window.open('https://github.com/EddieJaoudeCommunity/EddieBot','_blank');"
         >
           <div
```

Comment:
I think you misplaced the `onclick` event in line **54**, it should be on the `div` in line *56*